src/css/application.css

```diff
@@ -420,13 +421,13 @@ body {
 }
 
 .editors__column-divider {
-  background-color: var(--color-dark-gray);
+  background-color: #aaa;
```

Comment:
Wow totally forgot about this... any chance you have thoughts on this @outoftime ?